README.md

```diff
@@ -95,6 +95,6 @@ repeat these steps to upgrade to future releases.
 ## Building
 
 * [Linux](docs/build-instructions/linux.md)
-* [OS X](docs/build-instructions/os-x.md)
+* [macOS](docs/build-instructions/macos.md)
```

Comment:
@paulcbetts While I can agree in general, is it really a problem in this case? We should rename the file at some point regardless, and I say sooner rather than later.

My vote is to keep this rename. /cc @atom/feedback 

---
_As a side note, an external link directly to a file on a `master` branch from is a bad idea exactly because of reasons such as this. Always, always link to an explicit version/commit._